eng/pipelines/mirror-repos.yml

```diff
@@ -23,41 +23,48 @@ jobs:
               azure-sdk/azure-rest-api-specs:
 
           Azure/azure-sdk-for-go:
+            Branch: main
```

Comment:
I don't think we need this override. As we already updated the code to use the default branch if no branch is specified. Have a look at https://github.com/Azure/azure-sdk-tools/blob/master/eng/pipelines/templates/steps/sync-repo-branch.yml#L38

Reply:
Good point, closing.